examples/aws/packages/lambda-outbound-proxy/index.js

```diff
@@ -0,0 +1,24 @@
+const handler = async () => {
```

Comment:
maybe use `export const handler = ...` instead of `exports.handler`?

Reply:
Mhh, we use this syntax in our other js examples as well (response-header, simple). Want to rather keep it uniform.